routes/nodebb.js

```diff
@@ -34,6 +34,7 @@ app.post(`${BASE_REPORT_URL}/forum/v2/read`, proxyObject());
 app.post(`${BASE_REPORT_URL}/forum/v2/create`, proxyObject());
 app.post(`${BASE_REPORT_URL}/forum/v2/remove`, proxyObject());
 app.post(`${BASE_REPORT_URL}/forum/v3/create`, proxyObject());
+app.post(`${BASE_REPORT_URL}/context/tags`, proxyObject())
```

Comment:
What is mean by "context" tags?
Call it as forum/tags

Reply:
Modified the name